updatecli/values.yaml

```diff
@@ -0,0 +1,5 @@
+github:
```

Comment:
Now that I read this all the following parameters are already defined in your manifest so you can probably remove them. 
But I usually prefer to template all scm configuration so I can quickly override them to target a different repository.
For example, I can try this [manifest](https://github.com/civo/kubernetes-marketplace/blob/9cd77ddd86179c679f11be182ad9d72a52d89332/.github/updatecli/updatecli.d/rancher.yaml#L36) just by changing this line https://github.com/civo/kubernetes-marketplace/blob/9cd77ddd86179c679f11be182ad9d72a52d89332/.github/updatecli/values.d/scm.yaml#L2